src/app/components/about/about.js

```diff
@@ -0,0 +1,70 @@
+import Link from "next/link"
+
+export default function AboutSection() {
+    return (
+        <div >
+            <div >
+                <h1></h1>
+                <h3 className={`text-center mt-8 ml-10 mr-10 text-sky-300 text-4xl md:text-6xl  mb-2 mt-2`}>About Us</h3>
+                <p className={`ml-10 mr-10 font-sans flex flex-row mt-8 gap-2`} >
+                    The Hour of Code started as a one-hour introduction to computer science, designed to demystify "code", to show that anybody can learn the basics, and to broaden participation in the field of computer science. It has since become a worldwide effort to celebrate computer science, starting with 1-hour coding activities but expanding to all sorts of community efforts. Howard County's version of Hour of Code is made by students, for students.
+                    <br></br><br></br><br></br><br></br>
```

Comment:
just do this with margin or padding or gap or smth 😭

Reply:
done